jjb/core/core-consul.yaml

```diff
@@ -19,18 +19,30 @@
           docker_tag: 'master'
     jobs:
       - '{project-name}-{stream}-verify-docker'
-      - '{project-name}-{stream}-merge-docker'
-      - '{project-name}-{stream}-release-version-docker-daily-no-sonar'
+      - '{project-name}-{stream}-merge-docker':
+          custom_triggers:
+            - reverse:
+                jobs: '{project-name}-{stream}-merge-java'
```

Comment:
This job does not appear to exist. How can it use a reverse trigger?

Reply:
Possibly more clean up needed here. If you look at production this is the current state for these jobs like core-consul, edgex-mongo, etc. Since there is no upstream job anymore (it probably existed in the past) it hasn't triggered in the past 7+ months. My goal was to minimize the changes before others could see.

The only repo where the chains of jobs all exist is support-rulesengine so at the very least support-rulesengine will need the old java triggers intact.

Reply:
@MightyNerdEric the jobs that don't exist is related to the cleanup of the legacy java repos and the removal of all of the legacy jobs that were related to those java builds.  This cleanup is necessary and was under the consult of @JPWKU

Reply:
consul-core is Archived